rust/kernel/src/printk.rs

```diff
@@ -58,15 +90,30 @@ impl fmt::Write for LogLineWriter {
 #[macro_export]
 macro_rules! println {
     () => ({
-        $crate::printk::printk("\n".as_bytes());
+        kprintln!(level: $crate::printk::KERN_INFO);
     });
-    ($fmt:expr) => ({
-        $crate::printk::printk(concat!($fmt, "\n").as_bytes());
+    ($msg:expr) => ({
+        kprintln!(level: $crate::printk::KERN_INFO, $msg);
```

Comment:
I don't love this syntax. Putting a "keyword argument" (which is what we're emulating) before positional arguments feels weird to me.

Reply:
Took that from the log crate. The do it with the target argument